iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -838,6 +845,9 @@ public void storageHandlerCommit(Properties commitProperties, Operation operatio
     }
     HiveIcebergOutputCommitter committer = getOutputCommitter();
     try {
+      if (context != null) {
+        committer.commitTask(context);
```

Comment:
what is that task and why it's in Iceberg SH?

Reply:
this happens in HS2, where we don't have the actual execution (tez) tasks
we simply write out iceberg data from HS2 from a single place

Reply:
if you are using writer directly why not call `commit` on iceberg table as well
````
icebergTable.newFastAppend().appendFile(file).commit()
````

Reply:
can't we expose `HiveIcebergOutputCommitter` via get method?
````
OutputCommitter outputCommiter = table.getStorageHandler().getOutputCommitter()
````

Reply:
exposing getOutputCommitter turned out to be a clean and working solution, as it was already public in HiveIcebergStorageHandler